pkg/model/constants.go

```diff
@@ -18,4 +18,7 @@ const (
 	PostgresqlPersistentVolumeCapacity = "1Gi"
 	DatabaseSecretUsernameProperty     = "user"
 	DatabaseSecretPasswordProperty     = "password"
+	KeycloakServicePort                = 8443
+	AdminUsernameProperty              = "SSO_ADMIN_USERNAME"
```

Comment:
Since this is a keycloak install, these need to be `KEYCLOAK_USER` and `KEYCLOAK_PASSWORD`. According to https://github.com/keycloak/keycloak-containers/tree/master/server#creating-admin-account

Reply:
Those are only the keys to pull the value from the secret. The actual env vars should have the correct names: https://github.com/keycloak/keycloak-operator/pull/39/files#diff-ae3bc997be08b0c72ba9b14bab077a2cR146

Reply:
thanks

Reply:
I think this part is already done in David's PR